src/components/CountryPicker/CountrySelectorModal.js

```diff
@@ -52,6 +52,7 @@ function CountrySelectorModal({currentCountry, isVisible, onClose, onCountrySele
                 keyForList: countryISO,
                 text: countryName,
                 isSelected: currentCountry === countryISO,
+                searchValue: countryName.toLowerCase().replaceAll(CONST.REGEX.COUNTRY_NAME_WITH_ONLY_ALPHABETS, ''),
```

Comment:
This isn't needed

Reply:
Apologies, nevermind, this **is** necessary
Removing it would break cases with countries that have special characters in their name

Reply:
> I added it here because if we store it in the state we won't have to recalculate it.

Fair enough, let's leave it here